osu.Game.Rulesets.Osu/Mods/OsuModHidden.cs

```diff
@@ -160,5 +164,15 @@ private void applyState(DrawableHitObject drawableObject, bool increaseVisibilit
                 }
             }
         }
+
+        private static void hideSpinnerApproachCircle(DrawableSpinner spinner)
+        {
+            var approachCircle = (spinner.Body.Drawable as IHasSpinnerApproachCircle)?.ApproachCircle;
+            if (approachCircle == null)
+                return;
+
+            using (spinner.BeginAbsoluteSequence(spinner.HitObject.StartTime - spinner.HitObject.TimePreempt))
+                approachCircle.Hide();
```

Comment:
What about other `IMutateApproachCircle` mods that also transform hitcircle approach circles? (`SpinIn`, `ObjectScaleTween`, `ApproachDifferent`)?

Reply:
All of these mods change nothing to the `DrawableSpinner`, so I've held up from hiding it just yet.